### ISSUE TITLE
It's work?

### ISSUE THREAD

**Issue by eleimt** [oguzcandemircan/laravel-stub-generator#1]
Hello.
It's with Lumen v8.*?

**Comment by oguzcandemircan** [comment]
Hi @eleimt,

I haven't tried it but I hope it works. I don't see any reason why it wouldn't work.

**Comment by eleimt** [comment]
composer say errors

**Comment by oguzcandemircan** [comment]
Are you getting an error like this?

![untitled](https://user-images.githubusercontent.com/19665071/125150330-a3163000-e147-11eb-8857-41b253ccfb41.PNG)

**Comment by oguzcandemircan** [comment]
If getting an error like this. I fixed the error #2 

try again:
```bash
composer require oguzcandemircan/laravel-stub-generator "v0.1.1"
```
Make sure you are using version v0.1.1
@eleimt

**Comment by eleimt** [comment]
Okay, I try. Thanks.
